web_external/js/views/widgets/WMSFeatureInfoWidget.js

```diff
@@ -25,24 +25,26 @@ minerva.views.WmsFeatureInfoWidget = minerva.View.extend({
         function getActiveGeojsonLayers() {
             var geojsonLayers = [];
             _.chain(that.parentView.collection.models)
-                .filter(function (set) { return set.get('displayed') && set.getDatasetType() === 'geojson'; })
+                .filter(function (set) { return set.get('displayed') && set.get('visible') && (set.getDatasetType() !== 'geojson' || set.getDatasetType() !== 'geojson-timeseries') })
```

Comment:
`set.getDatasetType() !== 'geojson' || set.getDatasetType() !== 'geojson-timeseries'` will always be true.  I think you want `set.getDatasetType() === 'geojson' || set.getDatasetType() === 'geojson-timeseries'`